### PR TITLE
Linting

### PR DESCRIPTION
**Pull request by spiray**
@MathuraMG I created a branch called linting which i am working in for now. Can you close #70 because that was done in master. In this pr i will commit all eslintrc file changes and changes to the project as a result of the lint rules. I am actually having a challenge now with extending the p5js eslint ruleset but I am working on that. I also notice #56, do you want me to delete that file?

### CODE REVIEW

**Comment by spiray** [comment]
After linting all of the files all looks good besides for i am not understanding the syntax for these 2 files ([1](https://github.com/processing/p5.accessibility/blob/master/src/textInterceptor/interceptorFunctions.js), [2](https://github.com/processing/p5.accessibility/blob/master/src/gridInterceptor/interceptorFunctions.js)).
You are essentially declaring a prototypical class and then assigning an instance of that class to the same name...? Is the class supposed to be Pascal case?

File: src/gridInterceptor/interceptorFunctions.js [hunk 1]
```diff
@@ -1,147 +1,153 @@
...
+            });
+            elementDetail.appendChild(objectList);
+        }
     }
-  }
 }
 
 
-var gridInterceptor = new gridInterceptor();
+const gridInterceptor = new gridInterceptor();
```
Comment (lm-n) [inline src/gridInterceptor/interceptorFunctions.js]: Using Pascal case this shouldn't be an issue. Can you change it to Pascal case?

File: src/textInterceptor/interceptorFunctions.js [hunk 2]
```diff
@@ -1,210 +1,210 @@
...
+                    ` of the canvas`;
+            });
+            element.appendChild(objectList);
+        }
     }
-  }
 }
 
-var textInterceptor = new textInterceptor();
+const textInterceptor = new textInterceptor();
```
Comment (lm-n) [inline src/textInterceptor/interceptorFunctions.js]: Using Pascal case this shouldn't be an issue. Can you change it to Pascal case?

**Comment by MathuraMG** [comment]
@spiray 
Yes, textInterceptor is a prototype class and yes, also an instance with the same name (created in the last line)
You're right, we should change the className to PascalCase, but if it is a prototype function, it should be camelCase

File: src/gridInterceptor/interceptorFunctions.js (hunk 1 above)
Comment (spiray) [reply src/gridInterceptor/interceptorFunctions.js]: This has been changes in the last commit

**Comment by spiray** [comment]
I believe all linting is done now and the examples are not giving any errors. However, the non-styled example is just displaying the sketch and isn't outputting the accessibility part...? Let me know if you want to add any linting rules, i think i have a few more that i want to implement.

**Comment by lm-n** [comment]
@spiray great! Thanks for contributing to the project! 🥇 😸

**Pull request merged**